client/src/App.js

```diff
@@ -28,8 +29,18 @@ const theme = createMuiTheme({
   },
 });
 function App() {
+  let location = window.location;
+  let DisplayNavBar = location.pathname !== "/" &&
+    location.pathname !== "/auth-with-google" &&
+    location.pathname !== "/sign-up" &&
+    location.pathname !== "/login-in" &&
+    location.pathname !== "/profile_settings" &&
+    location.pathname !== "/availability" &&
+    location.pathname !== "/confirm" && <NavBar />;
```

Comment:
For maintainability and readability I think we can declare something like `displayNavBarPaths = ["/auth-with-google", so on...]`.
Then we can have `let DisplayNavBar = displayNavBarPaths.includes(location.pathname) && <NavBar />`

Reply:
fixed